### PR TITLE
feat(maintenance) Upgrade bootstrap to latest version

### PR DESCRIPTION
**Pull request by Fefaine**

### CODE REVIEW

**Comment by cyberwolf** [comment]
👍

File: resources/multisite_drupal_standard.make [hunk 1]
```diff
@@ -937,7 +937,7 @@ libraries[respond][download][url] = "https://raw.githubusercontent.com/scottjehl
 
 projects[bootstrap][type] = theme
 projects[bootstrap][download][type] = get
-projects[bootstrap][download][url] = http://ftp.drupal.org/files/projects/bootstrap-7.x-3.5.zip
+projects[bootstrap][download][url] = https://ftp.drupal.org/files/projects/bootstrap-7.x-3.8.zip
```
Comment (tatarbj) [inline resources/multisite_drupal_standard.make]: what's wrong with the standard format?
`projects[bootstrap][type] = theme`
`projects[bootstrap][version] = 3.8`
I've double checked it in a simple site.make and it works like a charm.

**Comment by tatarbj** [comment]
:+1:

**Comment by Fefaine** [comment]
:+1:

**Pull request merged**